velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMethods.java

```diff
@@ -0,0 +1,521 @@
+        return obj != null ? obj.toString().replaceAll("\"", "") : null;
+    }
+
+    @Override
+    public String parseJSON(String s, String s1) {
+        JsonObject element;
+        try {
+            element = (JsonObject) JsonParser.parseString(s).getAsJsonObject();
+        } catch (IllegalStateException e) {
+            return null;
+        }
+        JsonElement obj = element.get(s1);
+        return obj != null ? obj.toString().replaceAll("\"", "") : null;
+    }
+
+    @Override
+    public Boolean getBoolean(Object o, String s) {
+        return ((YamlFile) o).getBoolean(s);
+    }
+
+    @Override
+    public String getString(Object o, String s) {
+        return ((YamlFile) o).getString(s);
+    }
+
+    @Override
+    public Long getLong(Object o, String s) {
+        return ((YamlFile) o).getLong(s);
+    }
+
+    @Override
+    public Integer getInteger(Object o, String s) {
+        return ((YamlFile) o).getInt(s);
+    }
+
+    @Override
+    public List<String> getStringList(Object o, String s) {
+        return ((YamlFile) o).getStringList(s);
+    }
+
+    @Override
+    public boolean getBoolean(Object o, String s, boolean b) {
+        return ((YamlFile) o).getBoolean(s, b);
+    }
+
+    @Override
+    public String getString(Object o, String s, String s1) {
+        return ((YamlFile) o).getString(s, s1);
+    }
+
+    @Override
+    public long getLong(Object o, String s, long l) {
+        return ((YamlFile) o).getLong(s, l);
+    }
+
+    @Override
+    public int getInteger(Object o, String s, int i) {
+        return ((YamlFile) o).getInt(s, i);
+    }
+
+    @Override
+    public boolean contains(Object o, String s) {
+        return ((YamlFile) o).contains(s);
+    }
+
+    @Override
+    public String getFileName(Object o) {
+        return "[Only available in the Bukkit Version!]";
+    }
+
+    @Override
+    public void callPunishmentEvent(Punishment punishment) {
+        server.getEventManager().fireAndForget(new PunishmentEvent(punishment));
+    }
+
+    @Override
+    public void callRevokePunishmentEvent(Punishment punishment, boolean b) {
+        server.getEventManager().fireAndForget(new PunishmentEvent(punishment, b));
+    }
+
+    @Override
+    public boolean isOnlineMode() {
+        return server.getConfiguration().isOnlineMode();
+    }
+
+    @Override
+    public void notify(String s, List<String> list) {
+        server.getAllPlayers().forEach(player -> {
+            if (player.hasPermission(s)) {
+                list.forEach(str -> sendMessage(player, str));
+            }
+        });
+    }
+
+    @Override
+    public void log(String s) {
+        server.getConsoleCommandSource().sendMessage(LegacyComponentSerializer.legacyAmpersand().deserialize(s));
+    }
+
+    @Override
+    public boolean isUnitTesting() {
+        return false;
+    }
+
+    //Extracted from https://github.com/Bukkit/Bukkit/blob/master/src/main/java/org/bukkit/plugin/java/JavaPlugin.java
```

Comment:
We are not in a Bukkit or similar environment and Velocity has Configurate available as a configuration library, there is no need to copy obsolete code from another platform (which could also be improved)